doc-content/enterprise-only/cases/case-management-install-and-login-to-showcase-proc.adoc

```diff
@@ -31,27 +31,27 @@ ifdef::DM[]
 endif::DM[]
 file. The
 ifdef::PAM[]
-(`rhpam-7.3-case-mgmt-showcase-eap7-deployable.zip`).
+(`rhpam-{PRODUCT_VERSION}-case-mgmt-showcase-eap7-deployable.zip`).
 endif::PAM[]
 ifdef::DM[]
-(`rhdm-7.3-case-mgmt-showcase-eap7-deployable.zip`).
+(`rhdm-{PRODUCT_VERSION}-case-mgmt-showcase-eap7-deployable.zip`).
 endif::DM[]
  file is in the unzipped directory.
 
 . Extract the
 ifdef::PAM[]
- (`rhpam-7.3-case-mgmt-showcase-eap7-deployable.zip`).
+ (`rhpam-{PRODUCT_VERSION}-case-mgmt-showcase-eap7-deployable.zip`).
 endif::PAM[]
 ifdef::DM[]
- (`rhdm-7.3-case-mgmt-showcase-eap7-deployable.zip`).
+ (`rhdm-{PRODUCT_VERSION}-case-mgmt-showcase-eap7-deployable.zip`).
 endif::DM[]
  archive to a temporary directory. In the following examples this directory is called `__TEMP_DIR__`.
 . Copy the contents of the
 ifdef::PAM[]
-`_TEMP_DIR/rhpam-7.3-case-mgmt-showcase-eap7-deployable/jboss-eap-7.3`
+`_TEMP_DIR/rhpam-{PRODUCT_VERSION}-case-mgmt-showcase-eap7-deployable/jboss-eap-{PRODUCT_VERSION}`
 endif::PAM[]
 ifdef::DM[]
-`_TEMP_DIR/rhdm-7.3-case-mgmt-showcase-eap7-deployable/jboss-eap-7.3`
+`_TEMP_DIR/rhdm-{PRODUCT_VERSION}-case-mgmt-showcase-eap7-deployable/jboss-eap-{PRODUCT_VERSION}`
```

Comment:
Good catch Marian. I will fix that now.

Reply:
There is still `jboss-eap-{PRODUCT_VERSION}` specified. It should be `jboss-eap-{EAP_VERSION}` everywhere because we don't have the same version of EAP as we have the version of our product.